### PR TITLE
Run prettier on existing code

### PR DESCRIPTION
**Pull request by kevin-wangg**
To run the prettier on the client, navigate into the client folder and run
`npm run pretty`.
To run the prettier on the server, navigate into the server and run
`npm run pretty`.

### CODE REVIEW

**Review by Anand-Issac — APPROVED** [review]

File: client/package.json [hunk 1]
```diff
@@ -39,7 +40,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint . --ext .tsx --ext .ts --fix"
+    "lint": "eslint . --ext .tsx --ext .ts --fix",
+    "pretty": "prettier --write \"./**/*.{ts,tsx}\""
```
Comment (meganniu) [inline client/package.json]: Let's format  css/scss files too
```
prettier --write "./**/*.{ts,tsx,scss,css}"
```

Also, can we add prettier to the server?

**Review by meganniu — CHANGES_REQUESTED** [review]
love the new styling @kevinwang2040 :')

File: client/package.json (hunk 1 above)
Comment (kevin-wangg) [reply client/package.json]: Added.

**Review by guy-ro — APPROVED** [review]
LGTM, so pretty!

**Review by Puepis — APPROVED** [review]
LGTM!

**Pull request closed without merging**